appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -5181,4 +5181,45 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
       "<br/>These applications will not run on Android versions older than 4.0.")
   @Description("Text for the Package SDK 26 Warning Dialog Box (HTML)")
   String Package26Notice();
+
```

Comment:
These should all be handled by AutogeneratedOdeMessages now. Do you still need them here?

Reply:
I added them because of habit. In Kodular, we still use OdeMessages (as we like using spaces in names), so I added them here just in case.